clio-reporting/clio-reporting-rest/src/main/java/eu/europeana/clio/reporting/rest/config/ApplicationConfiguration.java

```diff
@@ -1,34 +1,106 @@
 package eu.europeana.clio.reporting.rest.config;
 
-import eu.europeana.clio.reporting.core.ReportingEngine;
-import eu.europeana.clio.reporting.core.config.ConfigurationPropertiesHolder;
+import eu.europeana.clio.common.config.properties.ReportingEngineConfigurationProperties;
+import eu.europeana.clio.common.persistence.model.*;
```

Comment:
Please extend this import

Reply:
Absolutely correct, I guess I didn't properly setup my code editor formatter